README.md

```diff
@@ -44,14 +44,15 @@ helm uninstall apisix ./chart/apisix -n default
 
 1. How to install APISIX only?
 
-The Charts will install etcd 3.4.14 by default. If you want to install Apache APISIX only, please set `etcd.enabled=false` and set `etcd.host=http://your_etcd_address:2379`.
+The Charts will install etcd 3.4.14 by default. If you want to install Apache APISIX only, please set `etcd.enabled=false` and set `etcd.host={http://your_etcd_address:2379}`.
```

Comment:
Strange, if i don't want to install etcd, why should i still care and set the etcd host?

Reply:
We can modify the `enable` extension, its meaning is not to enable etcd, but switch configuration options.